sympy/stats/rv.py

```diff
@@ -561,6 +562,9 @@ def expectation(expr, condition=None, numsamples=None, evaluate=True, **kwargs):
 
     >>> E(X, X > 3) # Expectation of X given that it is above 3
     5
+    >>> X = Uniform('X', 0, 1)
+    >>> E(X, X > Rational(1,2), evaluate=True) == Rational(3,4)
+    True
```

Comment:
This could rather be `test_issue_13324` in sympy/stats/tests/test_continuous_rv.py.
Is `evaluate=True` necessary?